Automattic/wp-calypso
# Added domain volume information to to the receipt

Related to # https://github.com/Automattic/martech/issues/2986

## Proposed Changes

* Add domain volume to clients' receipts that register domains for multiple years.


## Why are these changes being made?


* This lets the client know the number of years the domain was registered for by looking at the receipt.

## Testing Instructions



* Go to the live link. Register a domain for multiple years. 
* In purchases->billing, look for your receipt.
* Verify that the number of years the domain has been registered for is shown on the receipt.

<img width="652" alt="Screenshot 2024-06-06 at 11 51 56" src="https://github.com/Automattic/wp-calypso/assets/11347472/24ada4bb-63bc-406b-9261-36e0119aaec8">



## Pre-merge Checklist
